samwise.js

```diff
@@ -6,30 +6,30 @@
  */
```

Comment:
SCRIPT-Good job on updating the documentation along with the changes. It's very thorough and helpful!